nucypher/cli/actions.py

```diff
@@ -129,8 +173,13 @@ def load_seednodes(emitter,
         teacher_uris = list()
 
     for domain in network_domains:
+
+        # Static nodes from JSON file
+        static_nodes = load_static_nodes(domain=domain)
+        teacher_uris.extend(static_nodes)
+
+        # Hardcoded teachers from module
```

Comment:
If network domains is ever > 1, isn't the `for` loop on line 192 re-adding already added teacher nodes for subsequent domains? If so, that's a big bug that gets worse the higher the number of domains. It feels like the loop on line 192 should be outside of the `for domain in network_domains:` loop (Line 175).

Reply:
:heavy_check_mark: